aip/0132.md

```diff
@@ -131,8 +132,9 @@ request message **should** contain a `string order_by` field.
   `"foo,bar"`.
 - The default sorting order is ascending. To specify descending order for a
   field, users append a `" desc"` suffix; for example: `"foo desc, bar"`.
-- Redundant space characters in the syntax are insignificant. `"foo, bar desc"`
-  and `" foo , bar desc "` are equivalent.
+- Redundant space characters in the syntax are insignificant.
```

Comment:
Comment on the whole section - we should describe the expected ordering behavior if there isn't an ordering field. How much does an API need to specify? Can it specify that there's no ordering?

Reply:
My general feeling is that they do not have to specify anything. Certainly we do not ask them to do that now, and almost nobody does.

Reply:
Hmm. It feels like we ought to say *something* here, if only to tell API producers whether or not *they* need to say something.

Reply:
Discussed on video chat. Agreed and done.